packages/backend/migrations/20221012013413_create_grants_table.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('grants', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('authServerId').notNullable()
+    table.foreign('authServerId').references('authServers.id')
+    table.string('continueId').nullable()
+    table.string('continueToken').nullable()
+    table.string('accessToken').nullable().unique()
+    table.string('accessType').notNullable()
+    table.specificType('accessActions', 'text[]')
+
+    table.timestamp('expiresAt').nullable()
+
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
+    table.timestamp('updatedAt').defaultTo(knex.fn.now())
+
+    table.unique(['authServerId', 'accessType', 'accessActions'])
+  })
+}
```

Comment:
Is it sufficient to not store the entire access array of the grant?

Reply:
It is for now since the RS is only requesting read-all incoming payment grants, and the new `GrantService` only supports creating grants of a single access type.
Do we anticipate the RS requesting multi-type grants that we should consider future-proofing for?